build.gradle.kts

```diff
@@ -4,18 +4,15 @@ val VERSION_NAME: String by project
 val GROUP: String by project
 
 plugins {
-    kotlin("jvm") version libs.versions.kotlin.get()
+    kotlin("jvm") version libs.versions.kotlin.get() apply false
+    kotlin("multiplatform") version libs.versions.kotlin.get() apply false
```

Comment:
Plugin has nothing to do with multiplatform. Why it's added here? Because plugin runs on project environment mostly on JVM.

Reply:
Done 👍